wordpress.org/public_html/wp-content/plugins/theme-directory/tests/Theme_Header_Storage_Test.php

```
@@ -0,0 +1,201 @@
+
+	/**
+	 * Runs the fixture through the first-submission branch and returns the post.
+	 *
+	 * @return WP_Post
+	 */
+	protected function store_fixture(): WP_Post {
+		$upload             = new WPORG_Themes_Upload();
+		$upload->theme      = new WP_Theme( $this->stylesheet, $this->theme_root );
+		$upload->theme_slug = 'fixture-theme-' . uniqid();
+		$upload->author     = get_user_by( 'id', 1 );
+		$upload->theme_post = null;
```

PHPStan (2.2.8)

[error] Property WPORG_Themes_Upload::$theme_post (WP_Post) does not accept null.


(assign.propertyType)